docs/review.md

```diff
@@ -0,0 +1,579 @@
+# Introduction
+
+Welcome to the code review for the Chitter Challenge!  Again, don't worry - you are not expected to have all the answers. The following is a code-review scaffold for Chitter Challenge that you can follow if you want to.  These are common issues to look out for in this challenge - but you may decide to take your own route.  
+
+Either way we'd very much appreciate you submitting the form, even if it's just to say that you didn't use it :-)
+
+Please use this form to tick off where your reviewee has successfully has successfully incorporated these guidelines!  This form helps us get an overall picture of how the whole cohort is doing - it's not an assessment of an individual student.
+
+# Step 0: Checkout and Run tests
+
+Please checkout your reviewee's code and run their tests. Read the code and try and use the app through the web interface.  You can also experiment with the engine in IRB. How easy is it to understand the structure of their code? How readable is their code? Did you need to make any cognitive leaps to 'get it'?
+
+# Step 1:  How far did they get?
+
+* Features
+  * [ ] Sign up
+  * [ ] Log in
+  * [ ] Log out
+  * [ ] Peeping
+  * [ ] Listing Peeps
+  * [ ] Display time of peep
+
+* Bonus Features
+  * [ ] Replying to peeps
+  * [ ] CSS styling
+
+The relevance of the subsequent steps may depend on how far the reviewee got with their challenge
+
+# Step 2: Structure and supporting files
+
+## Installation Instructions in README
+
+Particularly now that we have a database involved, it becomes even more important to ensure that correct installation instructions are included in your readme so that other developers (and yourself in the future) know how to get set up with the application:
+
+```
+$ git clone https://github.com/tansaku/chitter_challenge
+$ bundle
+$ echo 'CREATE DATABASE chitter_development' | psql
```

Comment:
Shouldn't it be

```shell-script
  echo 'CREATE DATABASE chitter_development | psql postgres
```
?

I think on Unix `psql`'s default DB name == username so if you didn't configure it beforehand it will throw an error